### DISCUSSION TITLE
Does Rsync resume or retry if interrupted while archiving?

### DISCUSSION THREAD

**Discussion by nycken** [marcone/teslausb#867] [Q&A]
Hi,

I'm new to this project and very excited to get started. I was wondering about possible rsync issues and how they would be handled.

Here are a few scenarios that I could think of. What does rsync do after these scenarios happen?
- rsync server is unavailable before archiving starts
- rsync server goes down in the middle of archiving
- Tesla USB-power turns off in the middle of archiving

For the last one, I read in another Q&A (Advice on Zero W 2 cooling and powering) that the USB stays on with Sentry Mode On, but does this apply if Sentry Mode is disabled at Home? (I have a 2020 MY, so USB is NOT in glovebox)

Thanks in advance for any advice!

**Comment by marcone** [comment]
> Here are a few scenarios that I could think of. What does rsync do after these scenarios happen?
> 
> * rsync server is unavailable before archiving starts
> * rsync server goes down in the middle of archiving
> * Tesla USB-power turns off in the middle of archiving

Archiving only starts when the archive server is reachable, so scenario 1 shouldn't be an issue.
For scenario 2, archiving will stop with an error, but should restart/resume once the archive server is up and running again.
For scenario 3, archiving will resume when the Pi boots again and the archive server is reachable.
